src/pvrclient-argustv.cpp

```diff
@@ -1493,8 +1528,20 @@ PVR_ERROR cPVRClientArgusTV::SignalStatus(PVR_SIGNAL_STATUS &signalStatus)
 /** Record stream handling */
 bool cPVRClientArgusTV::OpenRecordedStream(const PVR_RECORDING &recinfo)
 {
-  XBMC->Log(LOG_DEBUG, "->OpenRecordedStream(%s)", recinfo.strStreamURL);
-  std::string UNCname = ToUNC(recinfo.strStreamURL);
+  XBMC->Log(LOG_DEBUG, "->OpenRecordedStream()");
```

Comment:
You could move this line after you have found the name instead of removing the name from the log entry. ;-)

This comment also applies to other places in your PR.

Reply:
Yeah, that change was made before i had worked out how i was going to handle the url. Just reducing all the failures to the core stuff. Ill fix it back up